yarn.lock

```diff
@@ -8494,10 +8494,13 @@ __metadata:
     "@backstage/cli-common": "workspace:^"
     "@backstage/errors": "workspace:^"
     "@manypkg/get-packages": ^1.1.3
-    "@microsoft/api-documenter": ^7.17.11
-    "@microsoft/api-extractor": ^7.23.0
-    "@microsoft/api-extractor-model": ^7.17.2
-    "@microsoft/tsdoc": 0.14.1
+    "@microsoft/api-documenter": ^7.19.26
+    "@microsoft/api-extractor": ^7.33.7
+    "@microsoft/api-extractor-model": ^7.25.3
+    "@microsoft/tsdoc": ^0.14.2
+    "@microsoft/tsdoc-config": ~0.16.1
```

Comment:
`@microsoft/api-extractor` is still depending on version `1.16.1` so need to mimic the version to not have tsc conflicts